ccinput/tests/test_gaussian.py

```diff
@@ -3782,3 +3782,149 @@ def test_unavailable_calc_type(self):
 
         with self.assertRaises(ImpossibleCalculation):
             self.generate_calculation(**params)
+    def test_sp_HF_CP_catch_Err1(self):
```

Comment:
Instead of `_catch_Err1`, you should put a couple words related to the error itself. Here, it would probably be `_extra_frag_spec` or something like that.